gitpg/israel/techinfo

```diff
@@ -0,0 +1,5 @@
+
+c, python
+
```

Comment:
not big deal, but in future changes, be careful with adding unnecessary empty lines to your code. This one doesn't hurt so we can merge